web/cobrands/highways/assets.js

```diff
@@ -154,7 +154,10 @@ function add_highways_warning(road_name) {
         .addClass('btn')
         .appendTo($radios);
     $radios.appendTo($warning);
-    $('.change_location').after($warning);
+    $warning.append('<button type="button" class="btn btn--block js-reporting-page--next">Continue</button>');
+
+    $('.js-reporting-page').first().before($warning);
+    $warning.nextAll('.js-reporting-page').removeClass('js-reporting-page--active');
```

Comment:
aha, because of my hard-coded " if ($page.data('pageName') === 'category')", when if HE box has appeared that's no longer true, so mob_ok click needs to go to "first" if it's the first that is selected, right, ok.